### PR TITLE
operate and tasklist docs review

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Related to https://github.com/camunda/developer-experience/issues/63.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
@johanwelgemoed Please see my inline comments for some clarification, and feel free to loop in additional reviewers!

**Comment by johanwelgemoed** [comment]
@christinaausley what is your urgency on this?

**Comment by christinaausley** [comment]
@johanwelgemoed Not urgent -- feel free to have a look following release next week.

File: docs/components/operate/operate-introduction.md [hunk 1]
```diff
@@ -18,14 +18,21 @@ dark: useBaseUrl('img/operate-introduction_dark.png'),
 />;
 -->
 
-In addition to providing visibility into active and completed process instances, Operate also makes it possible to carry out key operations such as resolving [incidents](./userguide/resolve-incidents-update-variables.md), and updating process instance variables.
+<!--- is there a reason why we have the comment above? --->
```
Comment (akeller) [inline docs/components/operate/operate-introduction.md:21]: We can remove this. If you really dig into the git history, it looks like this is an artifact from the original Operate docs that included an image that could toggle for light/dark mode. A relic of the past.

File: docs/components/operate/operate-introduction.md [hunk 2]
```diff
@@ -18,14 +18,21 @@ dark: useBaseUrl('img/operate-introduction_dark.png'),
...
 Learn how to use Operate to monitor process instances and more features in the [Operate user guide](/components/operate/userguide/basic-operate-navigation.md).
 
-Operate is also available for production use (with support) in the Camunda 8 offering.
-
-To try out Operate in Camunda 8, sign up [here](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral).
+## Availability
 
 Because Operate can be a helpful tool when getting started with Zeebe and building an initial proof of concept, we make it available under the [Operate trial license](https://camunda.com/legal/terms/cloud-terms-and-conditions/general-terms-and-conditions-for-the-operate-trial-version/). There are no restrictions under this license when it comes to the length of the evaluation period or the available feature set _as long as you use Operate in non-production environments only._
+
+Operate is available for production use (with support) in the Camunda 8 offering. [Try out Operate in Camunda 8](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral).
```
Comment (akeller) [inline docs/components/operate/operate-introduction.md:38]: Can you check with [#top-rapid-adoption-acquisition](https://camunda.slack.com/archives/C03NEE32Y2F) if they monitor this or want to monitor this? 

@aleksander-dytko I assume you don't track this, right?

File: docs/components/operate/userguide/updating-operate.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ title: Updating Operate
 description: Instructions on how to update your Operate
 ---
 
+<!--- Sooooo we have this file but I don't see it in the sidebar. Should this be added? --->
```
Comment (akeller) [inline docs/components/operate/userguide/updating-operate.md:7]: I would vote to delete it. If we do keep it, this is only relevant for SM customers and should be moved.

File: docs/components/operate/operate-introduction.md (hunk 2 above)
Comment (aleksander-dytko) [reply docs/components/operate/operate-introduction.md:38]: No, I don't

File: docs/components/operate/userguide/resolve-incidents-update-variables.md [hunk 4]
```diff
@@ -9,6 +9,8 @@ import TabItem from "@theme/TabItem";
 
 Every process instance created for the [`order-process.bpmn`](./assets/order-process.bpmn) process model requires an `orderValue` so the XOR gateway evaluation will happen properly.
 
+<!--- Maybe I'm totally just missing something here, but where was this model from? Is it in previous documentation? --->
+
```
Comment (johanwelgemoed) [inline docs/components/operate/userguide/resolve-incidents-update-variables.md:13]: From what I can see this has alway been here. It is the model that needs to be deployed and used by the examples.

File: docs/components/operate/userguide/resolve-incidents-update-variables.md [hunk 5]
```diff
@@ -119,6 +121,8 @@ We’ll publish a message that will be correlated with the instance, so we can a
 
 In the Operate interface, you should now observe the process instance has an <!-- FIXME: [“Incident”](/reference/incidents.html) --> incident, which means there’s a problem with process execution that must be fixed before the process instance can progress to the next step.
 
+<!--- What is the FIXME we have going on here? --->
```
Comment (johanwelgemoed) [inline docs/components/operate/userguide/resolve-incidents-update-variables.md:124]: It looks like it is supposed to reference https://docs.camunda.io/docs/components/concepts/incidents/ perhaps?

File: docs/components/tasklist/introduction-to-tasklist.md [hunk 6]
```diff
@@ -19,4 +21,11 @@ The GraphQL and REST APIs are currently available, but the GraphQL API will be d
 we recommend using the REST API to ensure long-term compatibility.
 :::
 
+<!--- Are there still plans for deprecation here? --->
```
Comment (marcosgvieira) [inline docs/components/tasklist/introduction-to-tasklist.md:24]: Yes, there is plans for deprecating GraphQL API (it already doesn't have the same functionalities that REST APIs)

File: docs/components/tasklist/userguide/starting-processes.md [hunk 7]
```diff
@@ -29,15 +27,22 @@ Tasklist will then wait for the process to be executed. If the process generates
 There could be multiple reasons why you are not seeing any process in the **Processes** tab:
 
 - There is no process deployed to your environment.
+
+<!--- Could we link to documentation on running a process here? --->
```
Comment (marcosgvieira) [inline docs/components/tasklist/userguide/starting-processes.md:31]: I didn't get, are you suggesting us to add a link to how to deploy a process on modeler?

File: docs/components/operate/operate-introduction.md (hunk 2 above)
Comment (akeller) [reply docs/components/operate/operate-introduction.md:24]: Thanks for starting the convo @christinaausley - https://camunda.slack.com/archives/C03NEE32Y2F/p1700570859840009

Let's change this URL to `https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral&utm_content=operate`

File: docs/components/operate/userguide/updating-operate.md (hunk 3 above)
Comment (christinaausley) [reply docs/components/operate/userguide/updating-operate.md:7]: LOL. Did some digging and found this content _is_ in SM and was never removed here. Deleting 🥳

**Comment by akeller** [comment]
@christinaausley as I'm reviewing, I see you still have HTML comments on these docs. Can you clean that up and/or open separate issues if you want to get this merged?

File: docs/components/tasklist/introduction-to-tasklist.md [hunk 8]
```diff
@@ -20,3 +20,8 @@ we recommend using the REST API to ensure long-term compatibility.
 :::
 
 Tasklist is also available for production use (with support) in the Camunda 8 offering. To try out Tasklist in Camunda 8, sign up [here](https://signup.camunda.com/accounts?utm_source=docs.camunda.io&utm_medium=referral).
+
+<!--- In the Operate overview, we not availability based on trial/professional use, is that relevant for Tasklist as well then? See below:
```
Comment (christinaausley) [inline docs/components/tasklist/introduction-to-tasklist.md:24]: @johanwelgemoed This is the final comment I have -- could you clarify for me?

Comment (akeller) [reply docs/components/tasklist/introduction-to-tasklist.md:24]: This is not relevant for Tasklist. 

Also, I'm not seeing that link on the overall terms list. @christinaausley can you follow up internally with legal to see if this trial license terminology is still valid or should be replaced with a different link?

**Review by akeller — DISMISSED** [review]

**Comment by akeller** [comment]
@christinaausley, I removed the deleted file from both sidebars files. I'll give this a 👍 and let it merge

**Review by akeller — APPROVED** [review]

**Pull request merged**